Dockerfile

```diff
@@ -0,0 +1,28 @@
+# Build stage
+ARG PYTHON_VERSION=3.8
+FROM python:${PYTHON_VERSION} AS base
+
+ENV PYTHONUNBUFFERED=1
+
+WORKDIR /app
+
+# Copy the application code into the container
+COPY . .
+
+# Runtime stage
+FROM python:${PYTHON_VERSION}-slim
+WORKDIR /app
+
+# Copy the entire /app directory from the build stage
+COPY --from=base /app /app
+
+# Install dependencies and run migrations
+RUN pip install --upgrade pip && \
+    pip install -r requirements.txt && \
+    python manage.py migrate
```

Comment:
Better run migration by another step. It create one more layer, what is bad, but gives a better vision of what step failed